worker/stemworker/pipelines/annular_mask.py

```diff
@@ -36,7 +33,12 @@ def execute(path=None, **params):
         files = files[offset:offset+files_per_rank]
 
     # Create local stem
-    reader = io.reader(files, version=io.FileVersion.VERSION1)
+    reader = io.reader(files, version=io.FileVersion.VERSION3)
```

Comment:
is `VERSION3` going to break the other datasets we are using during development?

Reply:
Yep, I can add a version parameter?